doc/modules/calibration.rst

```diff
@@ -77,8 +77,12 @@ non-parametric approach based on isotonic regression (:mod:`sklearn.isotonic`).
 Probability calibration should be done on new data not used for model fitting.
 The class :class:`CalibratedClassifierCV` uses a cross-validation generator and
 estimates for each split the model parameter on the train samples and the
-calibration of the test samples. The probabilities predicted for the
-folds are then averaged. Already fitted classifiers can be calibrated by
+calibration of the test samples, when the parameter `ensemble` is set to True (default).
+The probabilities predicted for the folds are then averaged.
+If the parameter `ensemble` is set to False, then the estimator is fit on the whole train
+data, and the cross-validation generator is used to get predictions
+(using :func:`cross_val_predict`) on which to determine calibration parameters.
```

Comment:
Maybe reverse the sentence and say that the classifier is fitted on the whole data *after* mentioning the cross_val_predict thing.

Otherwise, one might get slightly confused and think that we use the same data for the classifier and for the calibrator

That being said, I've been working on https://github.com/scikit-learn/scikit-learn/pull/16175 to improve this file. Maybe wait for this one to be merged (hopefully soon) before addressing this comment.